dangerfile.ts

```diff
@@ -16,7 +16,7 @@ if (packageChanged && !lockfileChanged) {
 const modifiedFiles = danger.git.modified_files.filter((path) => /\/src\/.+\.tsx?/.exec(path));
 
 // Check for console.log statements
-const statements = ['console.debug', 'console.log', 'console.warn', 'describe.only', 'test.only'];
```

Comment:
Was this removal intentional?

Reply:
Yes...  We just added this yesterday.  I was on the fence about `console.log` and all, and this PR triggered a bunch of false positives.

The "recipe" that we're currently using scans the entire file if you touch a single line.  So, in this case, I touched `deploy.ts` which has valid use of `console.log()` (inside the bot), and no way to escape it / ignore it.

Ideally, (1) Danger.js would only look at lines changed and (2) Danger.js would provide an escape hatch for "ignore".

That's low pri future work though.